ci/Dockerfile-nighthawk-test-server

```diff
@@ -0,0 +1,11 @@
+FROM frolvlad/alpine-glibc
```

Comment:
Suggest moving these to their own dedicated directory, e.g. `ci/docker`, `ci/images` or something

Reply:
Done